launch-week-august-2023.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: 'Launch Week'
+description: 'Join us for an exciting week filled with announcements, fun activities, and a chance to win prizes!'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/launch-week.jpg
```

Comment:
I'm not sure why, but I'm getting an error on this preview image. 

<img width="715" alt="CleanShot 2023-08-28 at 06 58 25@2x" src="https://github.com/xataio/mdx-blog/assets/25181823/312d2e93-67ea-4482-91dd-a86cd64cd21b">

Reply:
It's because the image is not yet in the `main` branch of the repo. You can see it if you change to `https://raw.githubusercontent.com/xataio/mdx-blog/launch-week-blog-post/images/launch-week.jpg`.